src/styles/blocks/stars.css

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  padding-right: 4px;
```

Comment:
It was a gap before, but I heard there is a problem with the linter test, there or somewhere in the future tasks. 
If you don't mind please explain that rule wider, because from my perspective I used padding for all cases, it seemed to me that almost no difference for positioning, plus no margin collapse, and I will see extra padding if I only add background-color